openedx/core/lib/log_utils.py

```diff
@@ -0,0 +1,36 @@
+"""
+Helper functions for logging.
+"""
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+def audit_log(name, **kwargs):
+    """DRY helper used to emit an INFO-level log message.
+
+    Messages logged with this function are used to construct an audit trail. Log messages
+    should be emitted immediately after the event they correspond to has occurred and, if
+    applicable, after the database has been updated. These log messages use a verbose
+    key-value pair syntax to make it easier to extract fields when parsing the application's
+    logs.
+
+    This function is variadic, accepting a variable number of keyword arguments.
+
+    Arguments:
+        name (str): The name of the message to log. For example, 'payment_received'.
+
+    Keyword Arguments:
+        Indefinite. Keyword arguments are strung together as comma-separated key-value
+        pairs ordered alphabetically by key in the resulting log message.
+
+    Returns:
+        None
+    """
+    # Joins sorted keyword argument keys and values with an "=", wraps each value
+    # in quotes, and separates each pair with a comma and a space.
+    payload = u', '.join(['{k}="{v}"'.format(k=k, v=v) for k, v in sorted(kwargs.items())])
+    message = u'{name}: {payload}'.format(name=name, payload=payload)
```

Comment:
Hmm.. if I move it to a constant, away from the `format` call, I think it makes it harder to read the code.  You'll need to go and read the constant defined in a different place and then elsewhere make sure the format call is correct.